### ISSUE TITLE
Game crashing when opening up the map (modded version)

### ISSUE THREAD

**Issue by Mister-Curious** [nipkownix/re4_tweaks#33]
Whenever I open the map the game crashes, but it should be noted that I have a modified version of the binary that has special instructions for the map. These instructions essentially tell the game to always load **map_obj1_07.MAP** file (which I have replaced with a custom file)

Here are the raw HEX edits made to the bio4.exe binary that make this change:
--------------
A2 04 76 C7 00 0F B7 82 AC 4F 00 00

Changed To:

A2 04 76 C7 00 B8 00 01 00 00 90 90

I'm not sure if you can check this out on your own and try to figure out how to fix this for me, but it would be a great help for my project as I would really like to preserve the functionality of my mod with your DLL. I asked my engineer to look at why the crash was happening but he could not find any causes. Perhaps you might have better luck.

**Comment by emoose** [comment]
Hmm, seems to crash for me when selecting map screen both with & without re4_tweaks DLL in place. Tried 1.0.6 & 1.1.0 too, no luck with either. Are you using anything else along with it (Raz0r DLL etc?)

**Comment by Mister-Curious** [comment]
Thank you so much for taking the time to look into this. 

Just using Raz0r's DLL with vanilla 1.0.6 exe works fine (without Raz0r DLL, same as you, crash). 
It will work fine with Raz0r's DLL, but as soon as I use your DLL along with it, things stop working.

 Can you try it with [Raz0r's Companion DLL?](https://www.nexusmods.com/residentevil4/mods/32?tab=files)

Cheers

**Comment by nipkownix** [comment]
Looks like it is some incompatibility with `RaiseInventoryAlloc`.
I don't see how we could be breaking his implementation, though.

**Comment by Mister-Curious** [comment]
@emoose @nipkownix yes, as I suspected it is a memory patch conflict with Raz0r's DLL, which I believe also has an SS_Pzzl inventory memory allocation. Setting **RaiseInventoryAlloc = False** in the winmm.ini config prevents the conflict. 

I don't see the need for a patch from your end if users are going to be using both DLLs as we can simply disable this feature for your DLL with the config. That being said, is there anything else the **RaiseInventoryAlloc** function does besides allowing highpoly models in the case? I wouldn't want to miss out on anything else your amazing DLL does.
